### PR TITLE
Add docs for terraform_remote_state

### PR DESCRIPTION
**Pull request by KFishner**

### CODE REVIEW

File: website/source/docs/modules/remote.html.markdown [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+other teams being able to build or modify that infrastructure.
+An example: a team builds and maintains a highly-available 
+database cluster, and other teams can access the URL and access 
+information via remote state without ever risking modifying that infrastructure.
+
+Remote states are accessed as a standard Terraform resource. An example is shown below:
+
+## An Example
+
+The `terraform_remote_state` resource first pulls information from an indepenedent
```
Comment (phinze) [inline website/source/docs/modules/remote.html.markdown]: s/information/outputs/

**Comment by mitchellh** [comment]
@c4milo The local backend is specifically just for development for now...

**Comment by mitchellh** [comment]
Done https://github.com/hashicorp/terraform/commit/56e92f7f021900dd030a0175018a4b66142752f7

**Comment by c4milo** [comment]
:+1:

**Pull request closed without merging**